frontend/app/src/features/questions/components/QuestionDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Text,
+  Box,
+  Button,
+  Tag,
+  Link,
+  HStack,
+  Wrap,
+  VStack,
+  Divider,
+} from "@chakra-ui/react";
+import { useQuestions } from "./QuestionsOutlet";
+import { useNavigate, useParams } from "react-router-dom";
+import { PiCaretLeftBold } from "react-icons/pi";
+import { Question } from "@/types/question";
+
+const QuestionDetails = () => {
+  const { data } = useQuestions();
+  const navigate = useNavigate();
+  const params = useParams();
+
+  const currQn: Partial<Question> | undefined = data?.data.questions.find(
+    (qn: Partial<Question>) => qn.id === parseInt(params.questionId as string),
+  );
+
+  return (
+    currQn && (
+      <>
+        <Button
+          leftIcon={<PiCaretLeftBold />}
+          variant="outline"
+          onClick={() => navigate(-1)}
```

Comment:
whats the behaviour of `navigate(-1)`?

Reply:
Go back to the previous page— I think this is now the standard practice for react-router v6